modules/swagger-codegen/src/main/resources/Baikal/newApiController.mustache

```diff
@@ -6,17 +6,19 @@ package {{package}};
 import play.mvc.Controller;
 import play.mvc.Result;
 import play.mvc.Http;
+import play.libs.Json;
+
 import java.util.List;
 import java.util.Map;
 import java.util.ArrayList;
-import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.google.inject.Inject;
 import java.io.IOException;
 import java.util.concurrent.CompletionStage;
-import java.util.concurrent.CompletableFuture;
 import swagger.SwaggerUtils;
+{{#isContainer}}
```

Comment:
It was giving unused imports. It is only necessary if you are using TypeReference class. You can see this on line 56. 

This is unrelated with the reason of the commit though.